### ISSUE TITLE
nuitka 2.5.6 cannot process transforms 4.46.2 packages 

### ISSUE THREAD

**Issue by hariji814** [Nuitka/Nuitka#3250]
Before submitting an Issue, please review the
[Issue Guidelines](https://github.com/Nuitka/Nuitka/blob/develop/CONTRIBUTING.md#submitting-an-issue).

- Please check whether the bug was already reported or fixed.

- Please check whether the issue appears when you run the code with Python, if it does, it's not a
  Nuitka issue.
nuitka 2.5.6 cannot process transforms 4.46.2 packages 
- Please check out if the develop version of Nuitka works better for you.
2.5.6
  Download source, packages [from here](https://nuitka.net/doc/download.html) where you will also
  find instructions how to do it via pip or git.

If you want to post a problem/bug, to help us understand and resolve your issue please check that
you have provided at least the information below, and discard up to here:

- Nuitka version, full Python version, flavor, OS, etc. as output by *this exact* command.

  > python -m nuitka --version
``` log
pip show transformers
Name: transformers
Version: 4.46.2
Summary: State-of-the-art Machine Learning for JAX, PyTorch and TensorFlow
Home-page: https://github.com/huggingface/transformers
Author: The Hugging Face team (past and future) with the help of all our contributors (https://github.com/huggingface/transformers/graphs/contributors)
Author-email: transformers@huggingface.co
License: Apache 2.0 License
Location: c:\users\zhangc\appdata\local\programs\python\python310\lib\site-packages
Requires: filelock, huggingface-hub, numpy, packaging, pyyaml, regex, requests, safetensors, tokenizers, tqdm
Required-by:
PS C:\Users\zhangc> python -m nuitka --version
2.5.6
Commercial: None
Python: 3.10.10 (tags/v3.10.10:aad5f6a, Feb  7 2023, 17:20:36) [MSC v.1929 64 bit (AMD64)]
Flavor: CPython Official
Executable: ~\AppData\Local\Programs\Python\Python310\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 10
Nuitka-Scons: Non downloaded winlibs-gcc 'D:\C++\mingw64\bin\gcc.exe' is being ignored, Nuitka is very dependent on the
Nuitka-Scons: precise one.
Version C compiler: ~\AppData\Local\Nuitka\Nuitka\Cache\downloads\gcc\x86_64\14.2.0posix-19.1.1-12.0.0-msvcrt-r2\mingw64\bin\gcc.exe (gcc 14.2.0).
```

  If you are not providing the full output. The issue cannot be solved and will be considered
  `invalid`. The command outputs more than you think, and we are adding more all the time. We do not
  want to left guessing or trying things out to reproduce your issue.

- How did you install Nuitka and Python
windows without conda
  Did you use pip, anaconda, deb, rpm, from source, git clone, then install into a virtualenv or
  not, this is very important usually and one thing, the above command does not tell us (yet).

- The specific PyPI names and versions

  It should be taken from this output if there specific packages involved, e.g. `numpy`, you are
  expected to shorten this to the relevant ones.

  > python -m pip list -v

- Many times when you get an error from Nuitka, your setup may be special

  Then even a `print("hello world")` program will not work, please try that and report that error
  instead. We do not need a report that says "Tensorflow does not work" when nothing actually works
  for you. You ought to also read the User Manual and check that your setup is actually supported
  already.

- Also supply a Short, Self Contained, Correct, Example

  That demonstrates the issue i.e a small piece of code which reproduces the issue and can be run
  with out any other (or as few as possible) external dependencies. Issues without this may will
  likely be rejected without much consideration. Often this can be as simple as importing a package,
  if this is a packaging issue, try that first.

  Pointers to repositories with usage of `pipenv` are very welcome, archives with examples are too,
  e.g. if a certain package structure is needed. This must be only source code, binaries are not
  used ever by us. But beware, that e.g. we cannot click around and stuff. Still do an effort to
  make the usage obvious. Having a compile script in the repo is perfect.

  But this cannot be much code for non-commercial users, since that causes too much effort. We
  cannot just compile your code, run it and have it download a control and command center.

- Provide in your issue the Nuitka options used

  Ideally use the `# nuitka-project:` options feature in the code, so options and example code go
  along. Alternatively state the command line.

  [Nuitka Options in the code](https://nuitka.net/doc/user-manual.html#nuitka-options-in-the-code)

- Avoid unnecessary options

  Do not use `--deployment` as it disables all potential bug catching, we are not here to replace
  what it does, remove that option the moment you face an issue and report only then.

  Do not use `--onefile` if the issue also happens with `--standalone`. Minimize the use of options
  as far as you can, please. Do not disable outputs with `--quiet` and do not disable warnings.

- Note if this is a regression

  If it used to work in an earlier version of Nuitka, please note what you know about that. Since
  git bisect is a thing for which we do a lot of error to make it usable, this will help
  dramatically to isolate the issue.

- Consider getting commercial support

  [Nuitka commercial](https://nuitka.net/doc/commercial.html) offers subscriptions and priority
  support. This will accelerate your problem solution and helps to sustain Nuitka development. Also
  you then have a chance to provide source code of your project to us, which might simplify things,
  or pay for time to solve your issues in your environment.

Some things are not welcome, please consider it.

- Do *not* post screenshots

  These are not welcome unless absolutely necessary, e.g. because of Qt display problem, instead
  capture the output of programs, so things are searchable and copy&paste will work. I just plainly
  don't want to manually copy strings and hope they match.

- Do *not* close the issue yourself, we will close things on stable releases

  We close issues only when they are released as a stable version, e.g. in a hotfix or a new
  release, before that it will be "Done" in planning and go through `factory` and `develop` tags to
  indicate they are solved there.

  Of course, if you find out your issue is invalid, please do close it, and we then attach the
  `invalid` tag.

- Do *not* report against factory version

  Unless you were asked to test it there, it is frequently very broken, and there is only noise to
  be had. Telling me about it on Discord would be a better idea.

- Do *not* let this template remain part of the issue, it's noise.

**Comment by hariji814** [comment]
1、my enviroment was 
2、my error log was
![Uploading 22222.png…]()

**Comment by hariji814** [comment]
the error is 
```log
F:\workSpace\cy-aipc-infer-sdk>F:\workSpace\cy-aipc-infer-sdk\o\start.dist\start.exe
Traceback (most recent call last):
  File "F:\workSpace\cy-aipc-infer-sdk\o\start.dist\transformers\utils\versions.py", line 102, in require_version
    got_ver = importlib.metadata.version(pkg)
  File "F:\WORKSP~1\CY-AIP~1\o\START~1.DIS\importlib\metadata\__init__.py", line 996, in version
  File "F:\WORKSP~1\CY-AIP~1\o\START~1.DIS\importlib\metadata\__init__.py", line 969, in distribution
  File "F:\WORKSP~1\CY-AIP~1\o\START~1.DIS\importlib\metadata\__init__.py", line 548, in from_name
importlib.metadata.PackageNotFoundError: No package metadata was found for tqdm

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "F:\WORKSP~1\CY-AIP~1\o\START~1.DIS\start.py", line 3, in <module>
  File "F:\WORKSP~1\CY-AIP~1\o\START~1.DIS\api\image_api.py", line 8, in <module api.image_api>
  File "F:\WORKSP~1\CY-AIP~1\o\START~1.DIS\command\video_wf_command.py", line 18, in <module command.video_wf_command>
  File "F:\WORKSP~1\CY-AIP~1\o\START~1.DIS\service\img_search_service.py", line 7, in <module service.img_search_service>
  File "F:\workSpace\cy-aipc-infer-sdk\o\start.dist\transformers\__init__.py", line 26, in <module>
    from . import dependency_versions_check
  File "F:\workSpace\cy-aipc-infer-sdk\o\start.dist\transformers\dependency_versions_check.py", line 57, in <module>
    require_version_core(deps[pkg])
  File "F:\workSpace\cy-aipc-infer-sdk\o\start.dist\transformers\utils\versions.py", line 117, in require_version_core
    return require_version(requirement, hint)
  File "F:\workSpace\cy-aipc-infer-sdk\o\start.dist\transformers\utils\versions.py", line 104, in require_version
    raise importlib.metadata.PackageNotFoundError(
importlib.metadata.PackageNotFoundError: No package metadata was found for The 'tqdm>=4.27' distribution was not found and is required by this application.
Try: `pip install transformers -U` or `pip install -e '.[dev]'` if you're working with git main

F:\workSpace\cy-aipc-infer-sdk>pip show transformers
WARNING: Package(s) not found: transformers
```

**Comment by hariji814** [comment]
i tried nuitka 2.6rtc  packge has some same question，what can i do？

**Comment by kayhayen** [comment]
I think `--include-distribution-metadata=tqdm` will help that. We already have a bunch of them in the Nuitka configuration.

If you are available to do a PR, this can be fixed by adding to the yaml configuration, described here: [https://nuitka.net/doc/nuitka-package-config.html](https://nuitka.net/doc/nuitka-package-config.html)

And here is more of a walk through style post: [https://nuitka.net/posts/nuitka-package-config-kickoff.html](https://nuitka.net/posts/nuitka-package-config-kickoff.html)

Here are a example PRs for other packages:

[#2030](https://github.com/Nuitka/Nuitka/pull/2030) 
[#1993](https://github.com/Nuitka/Nuitka/pull/1993)

**Comment by hariji814** [comment]
> I think `--include-distribution-metadata=tqdm` will help that. We already have a bunch of them in the Nuitka configuration.我认为 `--include-distribution-metadata=tqdm` 这会有所帮助。我们在 Nuitka 配置中已经有一堆 MissAV.com 浏览器。
> 
> If you are available to do a PR, this can be fixed by adding to the yaml configuration, described here: https://nuitka.net/doc/nuitka-package-config.html如果您可以执行 PR，则可以通过添加到 yaml 配置来解决此问题，如下所述：https://nuitka.net/doc/nuitka-package-config.html
> 
> And here is more of a walk through style post: https://nuitka.net/posts/nuitka-package-config-kickoff.html这里有一个更多的 style through 帖子：https://nuitka.net/posts/nuitka-package-config-kickoff.html
> 
> Here are a example PRs for other packages:以下是其他软件包的示例 PR：
> 
> [#2030](https://github.com/Nuitka/Nuitka/pull/2030) [#1993](https://github.com/Nuitka/Nuitka/pull/1993)

thank you ，master。i have slove that。have a good day！

**Comment by kayhayen** [comment]
I am guessing that means, it is the only change needed. I have added it to factory in hopes of that being true and will make it part of the next hotfix.

**Comment by kayhayen** [comment]
Part of the 2.5.7 release I just made.
